pkg/cli/workflow/list_runs.go

```diff
@@ -0,0 +1,119 @@
+package workflow
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jonboulle/clockwork"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/formatted"
```

Comment:
It could be risky to include a sub-package package from the Tekton CLI, given that it is not a public library and no guarantee that the exposed interface will change from version to version in a backwards-compatible way. But I really like what you've done with it and I think it's worth the risk. Great idea !